shade-memcached/src/main/scala/shade/memcached/Codec.scala

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) 2012-2017 by its authors. Some rights reserved.
+ * See the project homepage at: https://github.com/monix/shade
+ *
+ * Licensed under the MIT License (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy
+ * of the License at:
+ *
+ * https://github.com/monix/shade/blob/master/LICENSE.txt
+ */
+
+package shade.memcached
```

Comment:
I suppose going forward `Codec`s will be strictly tied to being used with Memcached?